src/Umbraco.Web/PublishedCache/NuCache/PublishedContent.cs

```diff
@@ -178,7 +178,7 @@ public override string Name
         {
             get
             {
-                if (!ContentType.Variations.Has(ContentVariation.CultureNeutral)) // fixme CultureSegment?
+                if (!ContentType.VariesByCulture())
                     return _contentData.Name;
 
                 var culture = VariationContextAccessor.VariationContext.Culture;
```

Comment:
There are no null checks here but these parts can be null, i've fixed in rev 7250b3d49198e908facde316d61f7fb8d0a0835f but see my comment in the task. I've changed to `VariationContextAccessor?.VariationContext?.Culture ?? ""`